MastodonSDK/Sources/MastodonUI/View/Content/StatusView+ViewModel.swift

```diff
@@ -283,6 +283,21 @@ extension StatusView.ViewModel {
                 authorView.dateLabel.configure(content: PlaintextMetaContent(string: text))
             }
             .store(in: &disposeBag)
+
+        $visibility
+            .map {
+                switch $0 {
```

Comment:
I feel like this would be better placed as a public extension so it can be shared with the compose view

Reply:
That was also my intention at first, but there are two different visibility enums. The status view uses `MastodonVisibility` and the compose view uses `Mastodon.Entity.Status.Visibility`. I can put this into in an extension, but it cannot be shared with the compose view unless the same enum is used in both cases.

Reply:
Haha right, I had forgotten about that. I think I merged them in one of my open PRs, but it hasn't been merged yet.